src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
It's good practice to avoid using wildcard imports. Instead, import only the classes you need.